library/templates/dcp_cpdb.yml

```diff
@@ -0,0 +1,38 @@
+dataset:
+  name: &name dcp_cpdb
+  version: "{{ version }}"
+  acl: private 
+  
+  source:
+    url:
+      path: library/tmp/cpdb/cpdb_dcpattributes_poly_18.zip
+      subpath: ""
+
+    geometry:
+      SRS: null
+      type: NONE
+
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES"
+
+
+  destination: 
+    name: *name
+
+    geometry:
+      SRS: null
+      type: NONE
+
+    options: 
+      - "OVERWRITE=YES"
+      - "PRECISION=NO"
+
+    fields: []
+    sql: null 
+
+  info:
+    description: |
+      Checkbook NYC Spending API: https://www.checkbooknyc.com/spending-api
```

Comment:
I would include a more detailed description, maybe also without the Checkbook NYC link (since this YAML is just for the CPDB geometries). We could include a brief overview of what CPDB is and what kind of data this is (points and polygons)